### PR TITLE
Add mutation type to the schema 

### PR DESCRIPTION
**Pull request by ani-sha**
Updating the schema, adding CRUD capabilities in ref to #16

### CODE REVIEW

**Comment by ani-sha** [comment]
For example : 
```
type MutationType {
  createAccount(input: CreateAccountInput!): ACCOUNT
  deleteAccount(input: MutateAccountInput!): ACCOUNT
  updateAccount(input: MutateAccountInput!): ACCOUNT
}

```

File: engine/src/main/java/io/graphqlcrud/types/JdbcTypeMap.java [hunk 1]
```diff
@@ -72,38 +66,58 @@ public GraphQLOutputType getAsGraphQLTypeString(int dataType) {
     }
 
     @Override
-    public GraphQLInputType getAsGraphQLFilterType (int dataType) {
-        GraphQLInputType outputType;
+    public GraphQLInputType getAsGraphQLTypeForInput(int dataType, String type) {
```
Comment (rareddy) [inline engine/src/main/java/io/graphqlcrud/types/JdbcTypeMap.java:69]: IMO you need to separate concerns of `mutation` and `filter` to different methods and not in single method

File: engine/src/main/java/io/graphqlcrud/GraphQLSchemaBuilder.java [hunk 2]
```diff
@@ -195,15 +257,35 @@ private void addQueryOperationsForEntity(Entity entity, Builder queryTypeBuilder
...
+        ArrayList<GraphQLInputObjectField.Builder> mutationFields = new ArrayList<>();
+
+        mutationEntity.getAttributes().forEach(attribute -> {
+            GraphQLInputObjectField.Builder builder = GraphQLInputObjectField.newInputObjectField();
+            builder.name(attribute.getName());
+            if (mutationEntity.isPartOfPrimaryKey(attribute.getName())) {
+                builder.type(GraphQLNonNull.nonNull(Scalars.GraphQLID));
+            } else {
+                if (attribute.isNullable()) {
+                    builder.type(TYPE_MAP.getAsGraphQLTypeForInput(attribute.getType(), "mutation"));
```
Comment (rareddy) [inline engine/src/main/java/io/graphqlcrud/GraphQLSchemaBuilder.java:282]: Looks like there is `filter` input type on `update` and should be there also on `delete`

Comment (ani-sha) [reply engine/src/main/java/io/graphqlcrud/GraphQLSchemaBuilder.java:282]: > Looks like there is `filter` input type on `update` and should be there also on `delete`

`filter input type`? Do you mean separate `input type` for delete? 

For both update and delete we have the same `Mutate<tablename>Input` input type.

Comment (rareddy) [reply engine/src/main/java/io/graphqlcrud/GraphQLSchemaBuilder.java:282]: https://graphqlcrud.org/docs/next/update#conditional-updates

**Comment by rareddy** [comment]
can you fix the test errors

**Comment by ani-sha** [comment]
> can you fix the test errors

I have been looking into this, locally the build passes. 
Only a single test case fails here.

**Review by rareddy — APPROVED** [review]

**Pull request merged**